### DISCUSSION TITLE
Start the fluidsynth daemon when an application requests midi?

### DISCUSSION THREAD

**Discussion by Bitwolfies** [FluidSynth/fluidsynth#1279] [Q&A] [answered]
Just curious if there's a way to automate the starting and closing of the fluidsynth daemon when an application (say, wine) requests to use midi.

**Comment by derselbst** [comment, marked as answer]
Most people just run fluidsynth as a systemd user daemon, so that applications like wine can use it when they need. There is not support for an "on-demand" starting / stopping of the daemon natively, i.e. you'd need to implement this by yourself.
